scripts/customer-info/customer_info.yaml

```diff
@@ -0,0 +1,224 @@
+bazel version: bazel 5.2.0
+bazel_action_information:
+- &id001
+  'Aspects:':
+    //java/...:
+    - '  BazelJavaProtoAspect: 3'
+    - ''
+  'Configurations:':
```

Comment:
I'd be really curious to see what this section looks like when the script is ran in another repo; details like `darwin_amd64` in `Execution Platforms` are _very_ helpful but the `Configurations` section seems a bit redundant in this example.

Reply:
I ran it on the engflow directory and this is what it looked like 
<img width="861" alt="Screen Shot 2023-01-04 at 10 20 51 AM" src="https://user-images.githubusercontent.com/61799142/210623090-da865cde-926d-4c92-a02c-efcc78406144.png">